flight/PiOS/STM32/pios_servo.c

```diff
@@ -99,6 +99,8 @@ static bool resetting;
 
 static bool dshot_in_use;
 
+static int telemreq_servo;
```

Comment:
At the least this needs to be volatile, but it probably needs to be atomic in actuality.  This also deepens the debt for flyingpi.